test/unit/test_runner.py

```diff
@@ -154,13 +155,9 @@ def test_stdout_file_write(rc, runner_mode):
     assert status == 'successful'
     stdout_path = Path(rc.artifact_dir) / 'stdout'
     # this can be subject to a race condition so we will be patient with the check
-    start_time = time.time()
-    while time.time() - start_time < 2.0:
+    for _ in iterate_timeout(2.0, 'stdout file to be written', interval=0.2):
```

Comment:
We can get some pretty slow test VMs, so 2 seconds can be cutting it pretty darn close, which is likely why you see the test failure. I see no reason not to just bump this to, say, 30 seconds.

Reply:
Thanks. The first time I checked results I saw 2 configurations failing and after adding the 2 second poll 1 was failing. So this would fit the picture that it's slow, dependent on the hardware.